app/views/includes/layout.html

```diff
@@ -1,28 +1,80 @@
-{% extends "views/layouts/govuk_template.html" %}
+{% extends "govuk/template.njk" %}
+{% from "govuk/components/phase-banner/macro.njk" import govukPhaseBanner %}
 
 {% block head %}
-  {% include "includes/head.html" %}
+  <link href="/public/stylesheets/application.css" rel="stylesheet">
+  <script src="/public/javascripts/ga.js"></script>
 {% endblock %}
 
-{% block cookie_message %}
-<p>{{__('GOV.UK uses cookies to make the site simpler.')}}
-  <a href="/cookies" title="{{__('Find out more about cookies')}}">{{__('Find out more about cookies')}}</a>
-</p>
+{# Temporarily using old-style cookie banner #}
+{% block bodyStart %}
+<div id="global-cookie-message">
+  <p>{{__('GOV.UK uses cookies to make the site simpler.')}}
+    <a href="/cookies" title="{{__('Find out more about cookies')}}">{{__('Find out more about cookies')}}</a>
+  </p>
+</div>
 {% endblock %}
 
-{% block header_class %}
-  with-proposition
+{%block beforeContent %}
+  {{ govukPhaseBanner({
+    tag: {
+      text: "beta"
+    },
+    html: 'This is a new service - your <a href="https://www.gov.uk/done/help-with-prison-visits">feedback</a> will help us to improve it. If you\'re having a problem, <a href="/help">get in touch</a>.'
+  }) }}
 {% endblock %}
 
-{% block footer_support_links %}
-<ul>
-  <li><a target="_blank" href="/cookies">Cookies</a></il>
-  <li><a target="_blank" href="privacy">Privacy</a></il>
-  <li><a target="_blank" href="https://www.gov.uk/done/help-with-prison-visits">Feedback</a></il>
-  <li><a target="_blank" href="/help">Technical help</a></li>
-</ul>
+{% block content %}
+  <h1 class="govuk-heading-xl">Default page template</h1>
 {% endblock %}
 
-{% block body_end %}
-  {% include "includes/scripts.html" %}
+{% block footer %}
+  {{ govukFooter({
+    meta: {
+      items: [
+        {
+          href: "/cookies",
+          text: "Cookies"
+        },
+        {
+          href: "/privacy",
+          text: "Privacy"
+        },
+        {
+          href: "https://www.gov.uk/done/help-with-prison-visits",
+          text: "Feedback"
+        },
+        {
+          href: "/help",
+          text: "Technical help"
+        }
+      ]
+    }
+  }) }}
+{% endblock %}
+
+{% block bodyEnd %}
+  {# Run JavaScript at end of the <body>, to avoid blocking the initial render. #}
+
+  {# scripts from old layout #}
```

Comment:
I will  be sorting out all these includes next: now everything is in new GDS it'll be clearer what's actually needed here...